vars/jobs.groovy

```diff
@@ -38,3 +38,13 @@ def build_job(name, parameters, is_stable)
   }
   return job
 }
+
+def build_and_notify(name, parameters, ignore_unstable=false){
+  def current_job = build job: name, parameters: parameters, propagate: false, wait: true
+  if (current_job.result != 'SUCCESS') {
+    if (env.NODE_NAME.contains('flow') || (ignore_unstable && current_job.result != 'UNSTABLE')) {
```

Comment:
still bad condition, it will be triggered always on node which contains flow, the || is or so you actually don't mind about rest if first is true. You will need to update condition wiht **and not**.